README.md

```diff
@@ -324,3 +324,13 @@ In order to comply with system shutdown, anyone should be ideally be able to red
 #### Graceful Shutdown
 
 If you are able to gracefully close out your module then we recommend you do so. An example of this is in [MIP50: Direct Deposit Module](https://github.com/makerdao/dss-direct-deposit/blob/master/src/DssDirectDepositAaveDai.sol#L287). You can see the permissionless `cage()` function which can be executed when `vat.live() == 0`. Upon execution, the system will begin gracefully closing itself out.
+
+### Gas Saving Tips
+
+Below are some random tips for saving gas that we have collected along the way:
+
+#### Use full-width types
+
+Intuitively you may think that by using two uint112s in storage right next to each other you are saving gas. This is not necessarily the case and in fact this pattern may slightly increase gas usage. Unless you are using both variables in the same code block you will still end with two SLOADs and further those SLOADs require bitwise masking operations to retrieve the desired piece.
```

Comment:
The example is fine with two `uint112`, but I think you might have wanted to write two `uint128`.
On the other hand, I think this case isn't actually that simple anymore. With the Berlin hard fork even if you load the two variables in different parts of the code it will be cheaper as the second SLOAD will cost much lower.
So in the only case that wouldn't make sense to pack variables that could be packed is if those variables are not being used in the same tx commonly.

Reply:
Good point. I'll make the changes.